doc/source/whatsnew/v1.0.0.rst

```diff
@@ -453,7 +453,7 @@ See :ref:`install.dependencies` and :ref:`install.optional_dependencies` for mor
 Other API changes
 ^^^^^^^^^^^^^^^^^
 
-- Bumpded the minimum supported version of ``s3fs`` from 0.0.8 to 0.3.0 (:issue:`28616`)
+- Bumped the minimum supported version of ``s3fs`` from 0.0.8 to 0.3.0 (:issue:`28616`)
```

Comment:
can you update the section that has our min supported version (in the whatsnew) (don't really need this issue line in that case)